index.bs

```diff
@@ -1233,7 +1233,7 @@ To <dfn>process cookie changes</dfn>, run the following steps:
     1. Let |url| be |window|'s [=creation URL=].
     1. Let |changes| be the [=observable changes=] for |url|.
     1. If |changes| [=set/is empty=], then [=continue=].
-    1. [=Queue a task=] to
+    1. [=Queue a global task=] to
```

Comment:
We probably need @domenic to review, but...

When specifying "global task" we don't need to pull out the event loop from window (that's what "global task" does, per https://html.spec.whatwg.org/multipage/webappapis.html#queue-a-global-task). We also want to follow the phrasing:

Queue a global task on XYZ task source given global to steps.

so e.g.

Queue a global task on XYZ task source given window to fire a change event named "change" with changes at window's CookieStore.

And then.. which task source? Storage and BroadcastChannel use the DOM manipulation task source so I assume it would be the same...? As @domenic asks on #159 we probably need to update the code. I'd go with DOM manipulation task source.

Reply:
Thanks for explaining @inexorabletash ! I've updated the wording and changed to use DOM manipulation task. 
Will ask domenic for additional review when he's back.